kitchen.recipes.yml

```diff
@@ -18,14 +19,15 @@ suites:
       # runs the recipe to test, along with the recipes it depends on
       - recipe[aws-parallelcluster::test_dummy]
       - recipe[aws-parallelcluster::example_recipe]
+      - recipe[aws-parallelcluster-install::sudoers]
       #- recipe[aws-parallelcluster-install::base]
-    verifier:
-      controls:
+    #verifier:
+      #controls:
         # verifier runs controls listed above, looking for them in test/recipes and test/resources
         # beware: these are not file names but control names
         # (see test/resources/example_control/controls/example_resource_spec.rb for an example)
         # this control tests the recipe
-        - example_recipe
+        #- example_recipe
```

Comment:
Please put your control here.

Reply:
This is not required, see previous comment.

Reply:
Anyway, added.